docs/src/components/landing/Hero.js

```diff
@@ -40,10 +41,20 @@ export default function Hero() {
           Drag and drop pre-built components, connect to data sources and APIs, and build your
           internal tools 10x faster. Open-source and powered by MUI.
         </Typography>
+        <Button
+          size="large"
+          variant="contained"
+          href="https://demo.toolpad.io/"
+          target="_blank"
```

Comment:
Maybe there's no need to open in a separate tab?
If the user tries the demo, do we want them to still use the landing page?

Reply:
I think the idea is that the user goes to the demo, returns to the landing page and then signs up for the beta if they liked the demo. Which is why a separate tab.

Reply:
Ok, sounds good in that case.